tests/bluetooth/controller/ctrl_feature_exchange/src/main.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (c) 2020 Demant
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <ztest.h>
+#include "kconfig.h"
+
+
+#define ULL_LLCP_UNITTEST
+
+
+#define TX_CTRL_BUF_NUM 2
+#define NTF_BUF_NUM 2
+
+#include <bluetooth/hci.h>
+#include <sys/byteorder.h>
+#include <sys/slist.h>
+#include <sys/util.h>
+#include "hal/ccm.h"
+
+#include "util/mem.h"
+#include "util/memq.h"
+
+#include "pdu.h"
+#include "ll.h"
+#include "ll_settings.h"
+
+#include "lll.h"
+#include "lll_conn.h"
+
+#include "ull_conn_types.h"
+#include "ull_tx_queue.h"
+#include "ull_llcp.h"
+#include "ull_llcp_internal.h"
+
+/* Implementation Under Test Begin */
+// #include "ll_sw/ull_llcp.c"
+/* Implementation Under Test End */
+
+#include "helper_pdu.h"
+#include "helper_util.h"
+#include "helper_features.h"
+
+struct ull_cp_conn conn;
+
+static void setup(void)
+{
+	test_setup(&conn);
+}
+
+
+/*
+ * +-----+                     +-------+            +-----+
+ * | UT  |                     | LL_A  |            | LT  |
+ * +-----+                     +-------+            +-----+
+ *    |                            |                   |
+ *    | Start                      |                   |
+ *    | Feature Exchange Proc.     |                   |
+ *    |--------------------------->|                   |
+ *    |                            |                   |
+ *    |                            | LL_FEATURE_REQ    |
+ *    |                            |------------------>|
+ *    |                            |                   |
+ *    |                            |    LL_FEATURE_RSP |
+ *    |                            |<------------------|
+ *    |                            |                   |
+ *    |     Feature Exchange Proc. |                   |
+ *    |                   Complete |                   |
+ *    |<---------------------------|                   |
+ *    |                            |                   |
+ */
+void test_feature_exchange_mas_loc(void)
```

Comment:
No control over FeatureSet_S and FeatureSet_M to verify FeatureSet_USED logic.

Reply:
Features are now passed in as parameter